### PR TITLE
E2-1936: add getvars

### PR DESCRIPTION
**Pull request by e7mac**
What              | Where/Who
------------------|----------------------------------------
JIRA Issue        | [E2-1936](https://leanplum.atlassian.net/browse/E2-1936)
People Involved   | @gg4race , @hrishileanplum 

[Notes on PR descriptions](https://leanplum.atlassian.net/wiki/spaces/PR/pages/288424408/Creating+a+GitHub+Pull+Request)

### CODE REVIEW

File: Leanplum/Api/LPGetVarsApi.m [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        failure(error);
+    };
+
+    NSMutableDictionary *params = [[NSMutableDictionary alloc] init];
+    if (parameters != nil) {
+        params = [parameters mutableCopy];
+    }
+    params[LP_PARAM_INCLUDE_DEFAULTS] = @(NO);
+    params[LP_PARAM_DEVICE_ID] = [LPAPIConfig sharedConfig].deviceId;
+    //TODO: what to do about below params
```
Comment (hrishileanplum) [inline Leanplum/Api/LPGetVarsApi.m]: Lets discuss this before we can approve this PR.

File: LeanplumTests/LPGetVarsApiTest.m [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    }];
+    
+    [self waitForExpectationsWithTimeout:10.0 handler:^(NSError *error) {
+        if (error) {
+            NSLog(@"Error: %@", error);
+        }
+    }];
+}
+
+//TODO : This is passing
```
Comment (hrishileanplum) [inline LeanplumTests/LPGetVarsApiTest.m]: ?

**Review by hrishileanplum — CHANGES_REQUESTED** [review]
Lets discuss the ToDo before approving.

File: LeanplumTests/LPGetVarsApiTest.m (hunk 2 above)
Comment (e7mac) [reply LeanplumTests/LPGetVarsApiTest.m]: so basically, there is no HTTP error, even though the dev key is empty

**Review by hrishileanplum — CHANGES_REQUESTED** [review]
We cannot merge this change, It does not handle models.

**Review by hrishileanplum — CHANGES_REQUESTED** [review]
We are not merging this PR yet.

**Pull request closed without merging**